Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityControllerMappingProfileInspector.cs

```diff
@@ -2,13 +2,13 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.﻿
```

Comment:
I'm seeing a couple bugs in this inspector, but didn't find the exact line for them.

Repro steps:
1. Scroll down to the generic Open VR Controllers. Do this on the **Right** Hand Controller, as there's a different bug preventing repro on the Left Hand Controller.
2. Try to change the Generic Type to Unity Controller while there's a Handedness selected. It won't change.
3. Switch the handedness to None, and switch to Unity Controller, successfully.
4. Change handedness to anything other than None, successfully.

The bug is either in step 2 or step 4 (or potentially both), depending on whether Unity Controllers support handedness, I suppose.

Reply:
Great catch, fixed.